common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -160,8 +161,10 @@ def test_render_student_view(self, view):
         self.assertIn("'prev_url': 'PrevSequential'", html)
         self.assertNotIn("fa fa-check-circle check-circle is-hidden", html)
 
+    # pylint: disable=line-too-long
     @patch('xmodule.seq_module.SequenceModule._get_user', return_value=UserFactory.build())
-    def test_timed_exam_gating_waffle_flag(self, mocked_user):
+    @patch('xmodule.seq_module.SequenceModule.gate_entire_sequence_if_it_is_a_timed_exam_and_contains_content_type_gated_problems')
+    def test_timed_exam_gating_waffle_flag(self, mocked_function, mocked_user):  # pylint: disable=unused-argument
```

Comment:
I think your arguments might be out of order here (worth a check maybe?). I thought the arguments mirror the order the patches are defined in so function and user would be flipped. Just mention because this would affect the other changes in this method.

Reply:
this always confuses me but I checked and believe what I have is correct. 
the first Note box here should explain it https://docs.python.org/3/library/unittest.mock.html#quick-guide